src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter((movie) => {
+    if (
+      movie.title.toLowerCase().includes(query.toString()
+        .toLowerCase()) || movie.description.toLowerCase()
+        .includes(query.toString().toLowerCase())
+    ) {
```

Comment:
let's make this code construction more readable (either create a variable, or use formatting, or a dedicated function – anything making it simpler to read)